service/src/test/java/bio/terra/pipelines/service/PipelinesServiceTest.java

```diff
@@ -330,4 +336,78 @@ void constructImputationInputsSuccess() {
     }
     assertEquals(totalInputs, allPipelineInputs.size());
   }
+
+  private static Stream<Arguments> mapInputTypeToCbasParameterTypeArguments() {
+    ParameterTypeDefinition stringParameterTypeResponse =
```

Comment:
feels like there ought to be a better way to test this, but trying to test the individual components of these objects got real messy real quick

Reply:
i think this is great